science/projects/Ghazaleh Mahmoudi/01-main-content/main_content_detection.ipynb

```diff
@@ -0,0 +1,1400 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #12.        description = soup.find('meta', {'property': 'og:description'})["content"]
</pre><p>please verify whether a meta description presents on the page or not.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Zarebin/nextgen-06/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>I call the create_page feature() function on the "try exception" in the prere_training_data() function, if the og:description is not displayed on the page, continue processed&nbsp;the next page.</p>